documentation/articles/xtoolkit/common.md

```diff
@@ -0,0 +1,98 @@
+# XToolkit Common
+
+The most common components without dependencies. Almost all projects ~~can~~ should use this library.
+
+## Install
+
+When you use this component separately from WhiteLabel.
+
+You can install via NuGet: [![Softeq.XToolkit.Common](https://buildstats.info/nuget/Softeq.XToolkit.Common?includePreReleases=true)](https://www.nuget.org/packages/Softeq.XToolkit.Common)
+
+```text
+Install-Package Softeq.XToolkit.Common
+```
+
+## Description
+
+### Common
+
+#### Collections
+
+Class | Description
+------|------------
+[ObservableRangeCollection](xref:Softeq.XToolkit.Common.Collections.ObservableRangeCollection`1) | Represents a dynamic data collection that provides notifications when items get added, removed, or when the whole list is refreshed.
+[ObservableItemContentRangeCollection](xref:Softeq.XToolkit.Common.Collections.ObservableItemContentRangeCollection`1) | Additionally to notifications inherited from ObservableRangeCollection notifies when any of the items (INotifyPropertyChanged) is updated.
+[ObservableKeyGroupsCollection](xref:Softeq.XToolkit.Common.Collections.IObservableKeyGroupsCollection`2) | Grouping of items by key into ObservableRange.
+[ObservableKeyGroupsCollectionNew](xref:Softeq.XToolkit.Common.Collections.ObservableKeyGroupsCollectionNew`2) | Represents groups of items that provides notifications when groups or it's items get added, removed, or when the whole list is refreshed.
```

Comment:
Sounds like it should be "... when groups or their items get added ..." instead of "... when groups or it's items get added ..."